docs/en/ingest-management/tab-widgets/prereq.asciidoc

```diff
@@ -11,12 +9,18 @@ all spaces.
 
 // tag::self-managed[]
 
-* Access to an {es} cluster and {kib} (version {minor-version}) with a basic
-license. {stack-gs}/get-started-elastic-stack.html[Learn how to install the
+* An {es} cluster and {kib} (version {minor-version}) with a basic license or
```

Comment:
Lists should be grammatically parallel, but sometimes it's not worth the effort because users don't really care (unless the list is harder to scan, which is a valid argument for making them parallel).